### DISCUSSION TITLE
Plug-ins and license files on separate computer from server?

### DISCUSSION THREAD

**Discussion by MDVolle** [apohl79/audiogridder#1324] [Q&A]
This is likely a really rookie question but all of my plug-ins are on an iMac Pro with ProTools - with an iLoc and licenses assigned to the iMac Pro.

I have an iMac i9 on the same desk I would like to use as a audiogridder server - how do I direct it to the existing plug-ins on the other machine or do I have to set up additional licensed copies on the server machine?

Both machines are on the same hard wired 1 gig network but could be connected directly via secondary network ports - would that improve performance or just use the existing network connection?

Ideally, it would be great to only maintain one set of instruments and plug-ins but if I need to set up copies, its probably possible for most of them.

Thanks

**Comment by Mike-Caffrey** [comment]
I think the DAW handles all of the licensing. You can have AAX, VST and AU plugins in different folders on a drive, but that's not a problem because the DAW handles the licensing. I'm pretty sure it's no different if you have the plugins on an external drive. 

Even with the processing happening on a different machine, there's no way for an unlicensed user to use them because their use is happening in the host DAW.
